tests/e2e/basic-ingest-redis-serving.py

```diff
@@ -23,31 +23,35 @@
 import uuid
 
 FLOAT_TOLERANCE = 0.00001
-PROJECT_NAME = 'basic_' + uuid.uuid4().hex.upper()[0:6]
+PROJECT_NAME = "basic_" + uuid.uuid4().hex.upper()[0:6]
+PROJECT_USER = "user_" + uuid.uuid4().hex.upper()[0:6]
 
-@pytest.fixture(scope='module')
+
+@pytest.fixture(scope="module")
 def core_url(pytestconfig):
     return pytestconfig.getoption("core_url")
 
 
-@pytest.fixture(scope='module')
+@pytest.fixture(scope="module")
 def serving_url(pytestconfig):
     return pytestconfig.getoption("serving_url")
 
 
-@pytest.fixture(scope='module')
+@pytest.fixture(scope="module")
 def allow_dirty(pytestconfig):
-    return True if pytestconfig.getoption(
-        "allow_dirty").lower() == "true" else False
+    return True if pytestconfig.getoption("allow_dirty").lower() == "true" else False
 
 
-@pytest.fixture(scope='module')
+@pytest.fixture(scope="module")
 def client(core_url, serving_url, allow_dirty):
     # Get client for core and serving
     client = Client(core_url=core_url, serving_url=serving_url)
     client.create_project(PROJECT_NAME)
```

Comment:
Where is the user setting their own identity (FEAST_USERNAME/FEAST_PASSWORD env vars)?

Reply:
We should probably seed the Core DB with a single admin user/pass in order to add more projects and members